ci/packages/package.go

```diff
@@ -341,21 +373,21 @@ func goGet(pkg string) error {
 	return sh.RunWith(map[string]string{"GO111MODULE": "off"}, "go", "get", "-u", pkg)
 }
 
-func packageStandalone(binaryPath, os, arch string) error {
+func packageStandalone(binaryPath, os, arch string, extraEnvs map[string]string) error {
```

Comment:
Consider wrapping all of these arguments into a struct if it's not too much work

Reply:
sure, no problems.
where do you suggest to declare the struct?

note: I'm a experience developer, but never touched golang before. So suggestions/tips related to golang patterns and good practices will always be wellcome

Reply:
> when you say "wrapping all of these arguments" you mean (binaryPath, os, arch string, extraEnvs map[string]string)? or something else?
where do you suggest to declare the struct?

Meant exactly this ☝️ 🙂